cornflow-server/cornflow/endpoints/execution.py

```diff
@@ -306,6 +307,33 @@ def _raise_af_error(execution, error, state=EXEC_STATE_UNKNOWN):
         execution.update_state(state)
         return execution, 200
 
+    @doc(description="Change status of an execution", tags=["Executions"])
+    @authenticate(auth_class=Auth())
+    def put(self, idx, **data):
```

Comment:
You could add here a:

``` python
@user_kwargs(StatusSchemaRequest)
@marshal_with(StatusSchemaResponse)
```

Si that you can check that what gets sent is correct and what get sent back as a response has a given structure as well.